### PR TITLE
refactor: use ButtonV2 in extension directories

### PR DESCRIPTION
**Pull request by kopancek**
### Describe what this PR does

refactored to use ButtonV2 in all of the files inside `packages/extension` directory

Related: https://dailydotdev.atlassian.net/browse/WT-2017

### Screenshots

Screenshot 2023-12-13 at 17 32 25
Screenshot 2023-12-13 at 17 32 34
Screenshot 2023-12-13 at 17 32 42
Screenshot 2023-12-13 at 17 34 09
Screenshot 2023-12-13 at 17 34 26

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [X] Have you done sanity checks in the extension?
- [X] Does this not break anything in companion?

### Did you test the modified components media queries?
- [X] MobileL (420px)
- [X] Tablet (656px)
- [X] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

### CODE REVIEW

File: packages/extension/src/companion/CompanionPopupButton.tsx [hunk 1]
```diff
@@ -61,12 +64,13 @@ export const CompanionPopupButton = (): ReactElement => {
...
+        variant={
           showCompanionPermission
-            ? 'btn-primary-cabbage'
-            : 'btn-secondary-cabbage',
-        )}
+            ? ButtonVariant.Primary
+            : ButtonVariant.Secondary
+        }
+        color={ButtonColor.Cabbage}
+        className="hidden laptop:flex mr-4 border-theme-status-cabbage"
```
Comment (nensidosari) [inline packages/extension/src/companion/CompanionPopupButton.tsx:73]: Was this something custom, `border-theme-status-cabbage` that the button didn't have by default or props?

**Review by nensidosari — COMMENTED** [review]
Looks great, just one small comment to check

File: packages/extension/src/companion/CompanionPopupButton.tsx (hunk 1 above)
Comment (kopancek) [reply packages/extension/src/companion/CompanionPopupButton.tsx:73]: For secondary button type, by default we show a white border on dark theme. This changes the border to the cabbage color. So it is custom only for this button.

**Review by nensidosari — APPROVED** [review]
Looks great 🥳

**Pull request merged**